pipeline/jobs/gradle-test.yaml

```diff
@@ -45,16 +45,20 @@ jobs:
   steps:
   - template: ../steps/run-gradlew.yaml
     parameters:
-      command: ${{ parameters.command }} -DENDPOINT_SUBSCRIPTION_KEY=$(subscriptionKey) -DENDPOINT_URL=https://${{ parameters.apimHost }} -DENDPOINT_HOST=${{ parameters.apimHost }}
+      command: ${{ parameters.command }} -DENDPOINT_SUBSCRIPTION_KEY=$(subscriptionKey) -DENDPOINT_URL=https://${{ parameters.apimHost }} -DENDPOINT_HOST=${{ parameters.apimHost }} -DMOCK_SERVER_HOST=localhost -DMOCK_SERVER_PORT=80
```

Comment:
Can we add the environment variable for logging as well - (API_LOG_LEVEL) ...a sample command is below.
./gradlew clean acceptance -DENDPOINT_URL=https://hmi-apim-svc-sbox.azure-api.net -DENDPOINT_SUBSCRIPTION_KEY=f4b0a75d51f8497da5dfb9e7de0d59f6 -DMOCK_SERVER_HOST=localhost -DMOCK_SERVER_PORT=80 -DAPI_LOG_LEVEL=DEBUG